### ISSUE TITLE
`no-empty` error false positives in empty blocks with comments

### ISSUE THREAD

**Issue by jfridye** [babel/babel-eslint#141]
The [eslint documentation for no-empty](http://eslint.org/docs/rules/no-empty) specifies the following is not considered a warning:

```
try {
    doSomething();
} catch (ex) {
    // Do nothing
}
```

yet with the `no-empty` rule enabled and  `eslint 0.24.0` and `babel-eslint 3.1.11` this results in a linting error.

**Comment by mathieumg** [comment]
This is a known Babel issue. See https://github.com/babel/babel/issues/1904 and https://github.com/babel/babel/issues/1486

**Comment by hzoo** [comment]
Oh haha. Yeah also the same as #124.
